### PR TITLE
docs: improve Supported Actions

### PR DESCRIPTION
**Pull request by jmfayard**
Hello, I optimized a bit how the "Supported Actions" look so that
- we have a direct link to the GitHub repo which has often very useful information
- we see the concrete Kotlin class name which is what we as Kotlin devs use

Supported_actions_-_GitHub_Actions_Kotlin_DSL

### CODE REVIEW

**Review by krzema12 — CHANGES_REQUESTED** [review]

File: docs/supported-actions.md [hunk 1]
```diff
@@ -6,92 +6,92 @@ Click on a version to see the wrapper's code.
 ## Wrappers
 
 * 10up
-    * action-wordpress-plugin-deploy ([v2.0.0](https://github.com/krzema12/github-actions-kotlin-dsl/tree/main/library/src/gen/kotlin/it/krzeminski/githubactions/actions/10up/ActionWordpressPluginDeployV2.kt))
+    *  [_GitHub_](https://github.com/10up/action-wordpress-plugin-deploy) - [ActionWordpressPluginDeployV2](https://github.com/krzema12/github-actions-kotlin-dsl/tree/main/library/src/gen/kotlin/it/krzeminski/githubactions/actions/10up/ActionWordpressPluginDeployV2.kt)
```
[the remark this answers is not in the document]
Comment (jmfayard) [reply docs/supported-actions.md]: That's good! I updated the format.

Supported_actions_-_GitHub_Actions_Kotlin_DSL

File: wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/GenerationEntryPoint.kt [hunk 2]
```diff
@@ -69,8 +69,9 @@ private fun generateListOfWrappersForDocs(listOfWrappersInDocs: Path) {
                 writer.println("* $owner")
                 ownedActions
                     .groupBy { it.name }
-                    .forEach { (action, versions) ->
-                        writer.println("    * $action (${versions.joinToString(", ") { it.toMarkdownLinkToKotlinCode() }})")
+                    .forEach { (_, versions) ->
+                        val kotlinClasses = versions.joinToString(", ") { it.toMarkdownLinkToKotlinCode() }
+                        writer.println("    *  ${versions.first().toMarkdownLinkGithub()} - $kotlinClasses")
```
Comment (krzema12) [inline wrapper-generator/src/main/kotlin/it/krzeminski/githubactions/wrappergenerator/GenerationEntryPoint.kt:74]: Shameless nitpick (but seriously, Markdown can interpret whitespaces in all weird ways, so it's better to be safe):
```suggestion
                        writer.println("    * ${versions.first().toMarkdownLinkGithub()} - $kotlinClasses")
```

**Review by krzema12 — CHANGES_REQUESTED** [review]
Only this one comment and we're merging :rocket:

**Review by krzema12 — APPROVED** [review]

**Comment by krzema12** [comment]
Thanks! :100:

**Pull request merged**